aikau/src/main/resources/alfresco/forms/controls/PushButtons.js

```diff
@@ -54,6 +56,28 @@ define(["alfresco/core/CoreWidgetProcessing",
       firstValueIsDefault: false,
 
       /**
+       * Extends the [inherited function]{@link module:alfresco/forms/controls/BaseFormControl#alfDisabled}
+       * to ensure that the [control]{@link module:alfresco/forms/controls/PushButtonControl} is disabled.
+       * 
+       * @instance
+       * @param  {boolean} status Whether or not the control should be disabled
+       * @since 1.0.80
+       */
+      alfDisabled: function alfresco_forms_controls_BaseFormControl__alfDisabled(status) {
+         this.inherited(arguments);
+
+         if (this.wrappedWidget && this.wrappedWidget.opts)
+         {
+            array.forEach(this.wrappedWidget.opts, function(option) {
```

Comment:
I've had this conversation with Martin before... we don't have to, but for the time being I want to keep the code consistent throughout the application. I'm also just in the habit of writing the code like this, so it's second nature.